vnet_peering.yml

```diff
@@ -15,12 +15,24 @@
   connection: local
   vars:
     resource_group: "{{ resource_group_name }}"
+    resource_group_secondary: "{{ resource_group_name }}2"
     vnet_name1: "myVnet{{ rpfx }}"
     vnet_name2: "myVnet{{ rpfx }}2"
+    peering_name: peer1
     location: eastus2
-  roles:
-    - Azure.azure_preview_modules    
+  # uncomment role if you're using ansible version < 2.8.
+  # At same install, install rule by ansible-galaxy install Azure.azure_preview_modules
```

Comment:
Needs rewording, and let's use "azure.azure_preview_modules" lowercase.
Let me think for a while what's the best wording.....